scene/gui/popup_menu.cpp

```diff
@@ -276,6 +276,7 @@ Size2 PopupMenu::_get_contents_minimum_size() const {
 		}
 	}
 
+	minsize.height = minsize.ceil().height; // Ensures enough height at fractional content scales to prevent the v_scroll_bar from showing.
```

Comment:
Nitpick, but isn't it enough to just ceil the y component?
```suggestion
	// Ensures enough height at fractional content scales to prevent the v_scroll_bar from showing.
	minsize.height = Math::ceil(minsize.height);
```

Reply:
Not a nitpick - it's a good suggestion! Will do...